File: sangjin/level1/28_시저_암호.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s, int n) {
    string answer = "";

    for(int i = 0; i < s.size(); i++){
        if(s[i] != ' '){
            if(s[i] <= 'Z'){
                s[i] = (s[i] - 'A' + n) % 26  + 'A' ;
            }
            else{
                s[i] = (s[i] - 'a' + n) % 26  + 'a' ;
            }
        }
    }

    return s;
}

File: yunj/level1/28_시저암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        char[] arr = s.toCharArray();
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] >= 97) {
                arr[i] = (char) (97 + (arr[i] - 97 + n) % 26);
            } else if (arr[i] >= 65) {
                arr[i] = (char) (65 + (arr[i] - 65 + n) % 26);
            }
        }
        answer = new String(arr);
        return answer;
    }
}